Gemfile.lock

```diff
@@ -696,5 +703,8 @@ DEPENDENCIES
   wunderground (~> 1.2.0)
   xmpp4r (~> 0.5.6)
 
+RUBY VERSION
```

Comment:
The docker image builds succeeded even though the ruby versions in the images do not match, so it might not. Maybe we should stop specifying the ruby version in the `Gemfile`as new bundler versions seem to write that version into `Gemfile.lock` http://bundler.io/whats_new.html?